client/src/assets/icons/green-not-completed.tsx

```diff
@@ -5,11 +5,15 @@ function GreenNotCompleted(
   props: JSX.IntrinsicAttributes & React.SVGProps<SVGSVGElement>
 ): JSX.Element {
   const { t } = useTranslation();
-
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+  const suppressSrOnly = props['data-suppress-sronly'];
```

Comment:
```suggestion
  const { suppressSrOnly, ...rest } = props;
```
Then, if you define an interface, `Props`, like so:
```ts
interface Props extends JSX.IntrinsicAttributes, React.SVGProps<SVGSVGElement> {
  suppressSrOnly: boolean;
}
```
you can spread `rest` over the svg.

Reply:
Thank you @ojeytonwilliams, the proper interface is what I was missing. I originally had this code the way you suggested but I couldn't figure out how to define the interface properly. That was the missing link!